src/frontend/apps/desk/src/features/members/components/ModalAddMembers.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button, Modal, ModalSize } from '@openfun/cunningham-react';
+import { useRouter } from 'next/router';
+import { useTranslation } from 'react-i18next';
+
+import { StyledLink } from '@/components';
+
+interface ModalAddMembersProps {
+  teamId: string;
+}
+
+export const ModalAddMembers = ({ teamId }: ModalAddMembersProps) => {
+  const { t } = useTranslation();
+  const router = useRouter();
+
+  return (
+    <Modal
```

Comment:
We might have already discussed it, but I would recommend closing the modal on click outside. This behavior is common on most of the web service, if we take a look at Trello GitHub and Drive invitation, all modals close on click outside, and recover state when necessary on re-open. I think for the v1 we can close the modal on click outside, and ignore the recovering state part. What did motivate this choice? Johann's design doesn't have any closing icon, I guess, he wanted the modal to be closed when clicking outside.

Reply:
https://github.com/numerique-gouv/people/pull/125/commits/f4e8b49b97b287412c6fc3021e654e83c618283f#diff-1c926ca4662a2efb0ebca0a591e1243d1322baaab47ee82337e65b73a66974eeR20-R21

[(app-desk) close modal role on click outside](https://github.com/numerique-gouv/people/pull/125/commits/04f9843bf8f9274e19837c8e223de390c4dff62d)